src/content/app/genome-browser/hooks/useBrowserRouting.test.tsx

```diff
@@ -52,6 +52,11 @@ jest.mock(
   })
 );
 
+jest.mock(
+  'src/content/app/genome-browser/hooks/useGenomeBrowserAnalytics',
+  () => () => jest.fn()
+);
```

Comment:
How is useBrowserRouting dependant on useGenomeBrowserAnalytics?

Reply:
Good questions. It shouldn't be.
I'm getting the following error when I remove the mock:

`Test suite failed to run

    TypeError: (0 , _rootReducer.default) is not a function

      33 | const epicMiddleware = createEpicMiddleware<Action, Action, RootState>();
      34 |
    > 35 | const rootReducer = createRootReducer();
         |                     ^
      36 |
      37 | const middleware = [
      38 |   epicMiddleware,

      at Object.<anonymous> (src/store.ts:35:21)
      at Object.<anonymous> (src/content/app/genome-browser/hooks/useGenomeBrowserIds.tsx:25:1)
      at Object.<anonymous> (src/content/app/genome-browser/hooks/useGenomeBrowserAnalytics.ts:21:1)
      at Object.<anonymous> (src/content/app/genome-browser/state/track-panel/trackPanelSlice.ts:25:1)
      at Object.<anonymous> (src/content/app/genome-browser/state/browser-general/browserGeneralSlice.ts:34:1)
      at Object.<anonymous> (src/content/app/genome-browser/state/genomeBrowserReducer.ts:19:1)
      at Object.<anonymous> (src/root/rootReducer.ts:19:1)
      at Object.<anonymous> (src/content/app/genome-browser/hooks/useBrowserRouting.test.tsx:26:1)
      at TestScheduler.scheduleTests (node_modules/@jest/core/build/TestScheduler.js:317:13)
      at runJest (node_modules/@jest/core/build/runJest.js:407:19)
      at _run10000 (node_modules/@jest/core/build/cli/index.js:338:7)
      at runCLI (node_modules/@jest/core/build/cli/index.js:190:3)`

I'll continue to investigate.

Reply:
This is now fixed. It was related to one of the Slices using the analytics hook.

Reply:
Slice using a hook? They shouldn't!